Components/Css/signup.css

```diff
@@ -0,0 +1,3 @@
+.login-form {
+    margin-left: 35rem;
+}
```

Comment:
you should use flexbox or grid to center the form, or at least use `margin: auto`.